hypertrace-gradle-docker-java-application-plugin/src/main/java/org/hypertrace/gradle/docker/HypertraceDockerJavaApplication.java

```diff
@@ -46,6 +48,10 @@ public HypertraceDockerJavaApplication(
                                     .convention(this.adminPort.map(adminPort -> String.format(
                                         "HEALTHCHECK --interval=2s --start-period=15s --timeout=2s CMD wget -qO- http://127.0.0.1:%d/health &> /dev/null || exit 1", adminPort)));
     this.orgLibrarySpec = this::isHypertraceLibrary;
+    // by default allow reflective access for monitoring executor service
+    // https://github.com/micrometer-metrics/micrometer/issues/2317#issuecomment-952700482
+    this.defaultJvmArgs = objectFactory.listProperty(String.class)
```

Comment:
We don't want/need this to be our own property, instead just piggy back off the existing `applicationDefaultJvmArgs` (the one you're setting in the plugin). That way, there aren't two ways to set the same thing.